src/lib/auth-utils.ts

```diff
@@ -169,6 +175,19 @@ export async function getApiToken(auth: IAuthWebUI, config: Config, remoteUser:
 
 }
 
+export async function getTokenKey(auth: IAuth, token: string): Promise<string> {
+  let key: string = "";
+  try {
+    // jwt token
+    const payload = (await verifyPayload(token, auth.secret, { ignoreNotBefore: true})) as RemoteUser&{jti?: string}
```

Comment:
Could you share more context about why we need `ignoreNotBefore` here?

Reply:
if a user sets the jwt `notBefore` configuration to 0 it will fail the jwt token validation step if it is action directly after the creation of the token, the ignore is an override to ignore the notBefore validation on creation of a new token which is required to extract the token id.